src/sidebar/helpers/account-id.js

```diff
@@ -31,17 +31,19 @@ export function username(user) {
 }
 
 /**
- * Returns true if the authority is of a 3rd party user.
+ * Returns true if the user's provider (authority) differs from the default
+ * authority for the application.
```

Comment:
Perhaps worth noting explicitly here that the default authority is the one that is considered first-party.